x-pack/legacy/plugins/siem/public/components/header_page/header_page.tsx

```diff
@@ -19,23 +23,59 @@ const Header = styled.header`
 
 Header.displayName = 'Header';
 
+interface DraggableArguments {
+  field: string;
+  value: string;
+}
+
 export interface HeaderPageProps {
+  'data-test-subj'?: string;
   badgeLabel?: string;
   badgeTooltip?: string;
   children?: React.ReactNode;
+  draggableArguments?: DraggableArguments;
   subtitle?: string | React.ReactNode;
   title: string | React.ReactNode;
-  'data-test-subj'?: string;
 }
 
 export const HeaderPage = pure<HeaderPageProps>(
-  ({ badgeLabel, badgeTooltip, children, subtitle, title, ...rest }) => (
+  ({ badgeLabel, badgeTooltip, children, draggableArguments, subtitle, title, ...rest }) => (
     <Header {...rest}>
       <EuiFlexGroup alignItems="center">
         <EuiFlexItem>
           <EuiTitle size="l">
             <h1 data-test-subj="page_headline_title">
-              {title}
+              {!draggableArguments ? (
+                title
+              ) : (
+                <DraggableWrapper
```

Comment:
Since we have a simple draggable here, we can replace the more flexable, but verbose `DraggableWrapper` with just a few lines of code via a `DefaultDraggable`:

```
                <DefaultDraggable
                  data-test-subj="header-page-headline-draggable"
                  id={`header-page-headline-draggable-${draggableArguments.field}-${draggableArguments.value}`}
                  field={draggableArguments.field}
                  value={`${draggableArguments.value}`}
                />
```